backend/test/acceptance/config.spec.js

```diff
@@ -36,4 +50,15 @@ describe('config', function () {
 
     expect(res.body).toMatchSnapshot()
   })
+
+  it('should reject unauthenticated requests', async function () {
```

Comment:
We do not need this test. The middleware of `/api` is already tested in an own acceptance test. Test only the good case.

Reply:
✅